### ISSUE TITLE
Fill Background Color not working

### ISSUE THREAD

**Issue by PolarisNation** [ClosedXML/ClosedXML#2097]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [x] Yes
- [ ] No

**Version of ClosedXML**
0.101.0

**What is the current behavior?**
The background does not set color.

**What is the expected behavior or new feature?**
I want the designated cell in Excel to be filled with color.

**Is this a regression from the previous version?**
No

## Reproducibility
**This is an important section. Read it carefully. Failure to do so will cause a 'RTFM' comment.**

Without a code sample, it is unlikely that your issue will get attention. Don't be lazy. Do the effort and assist the developers to reproduce your problem. Code samples should be [minimal complete and verifiable](https://stackoverflow.com/help/mcve). Sample spreadsheets should be attached whenever applicable. Remove sensitive information.

**Code to reproduce problem:**
```c#
               var workBook = new XLWorkbook();
               var workSheet = workBook.Worksheets.Add("Color Test");
               workSheet.Cell("A2").Value = "Code";
               workSheet.Cell("A2").Style.Fill.BackgroundColor = XLColor.Red;
               workSheet.Cell("A2").Style.Alignment.SetHorizontal(XLAlignmentHorizontalValues.Center);
               workBook.SaveAs("D:\\ColorTest.xlsx");

- [x] I attached a sample spreadsheet.  (You can drag files on to this issue)
[ColorTest.xlsx](https://github.com/ClosedXML/ClosedXML/files/11631796/ColorTest.xlsx)

**Comment by PolarisNation** [comment]
I think this look like https://github.com/ClosedXML/ClosedXML/issues/1861 this issue.
My Office version is 2016.
When I open the ColorTest Excel file I created in Office 2016, which is stored on my computer, I can't see the color, but when open it with Google Sheet, it looks normal.
Is there a part that sets the office version separately?
If possible, I guess it will be a background setting if I set it to 2016.
